src/parser.rs

```diff
@@ -1122,6 +1122,29 @@ impl Parser {
         })
     }
 
+    fn parse_column(&mut self) -> Result<ColumnDef, ParserError> {
```

Comment:
This is taken from `parse_columns`, right? Why not re-use it there now?

Reply:
Yes, this is taken from `parse_columns`, I can't re-use it , because in `alter table ` grammar, it doesn't start with `LParen`, if re-use `parse_columns`, it will always return an empty vector.
https://github.com/andygrove/sqlparser-rs/blob/fab6e2827195b6067cac5e5b3a1a9bb6bc161ae7/src/parser.rs#L1128

Reply:
I mean call the new `parse_column` method _from_ `parse_columns`, not the other way around.